test/k8sT/Policies.go

```diff
@@ -96,125 +95,130 @@ var _ = Describe("K8sPolicyTest", func() {
-		clusterIP, err := kubectl.Get("default", "svc").Filter(
+		clusterIP, err := kubectl.Get(helpers.DefaultNamespace, "svc").Filter(
 			"{.items[?(@.metadata.name == \"app1-service\")].spec.clusterIP}")
 		logger.Infof("PolicyRulesTest: cluster service ip '%s'", clusterIP)
 		Expect(err).Should(BeNil())
 
-		ciliumPod, err := kubectl.GetCiliumPodOnNode("kube-system", "k8s1")
+		ciliumPod, err := kubectl.GetCiliumPodOnNode(helpers.KubeSystemNamespace, helpers.K8s1)
 		Expect(err).Should(BeNil())
 
-		status := kubectl.CiliumExec(ciliumPod, "cilium config PolicyEnforcement=default")
+		status := kubectl.CiliumExec(ciliumPod, fmt.Sprintf("cilium config %s=%s", helpers.PolicyEnforcement, helpers.PolicyEnforcementDefault))
 		status.ExpectSuccess()
+
 		kubectl.CiliumEndpointWait(ciliumPod)
 
 		By("Testing L3/L4 rules")
 
 		eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
-		_, err = kubectl.CiliumImportPolicy("kube-system", l3Policy, 300)
+		_, err = kubectl.CiliumImportPolicy(helpers.KubeSystemNamespace, l3Policy, 300)
 		Expect(err).Should(BeNil())
 
 		err = waitUntilEndpointUpdates(ciliumPod, eps, 4)
 		Expect(err).Should(BeNil())
 		epsStatus := helpers.WithTimeout(func() bool {
-			endpoints, err := kubectl.CiliumEndpointsListByTag(ciliumPod, podFilter)
+			endpoints, err := kubectl.CiliumEndpointsListByLabel(ciliumPod, podFilter)
 			if err != nil {
 				return false
 			}
 			return endpoints.AreReady()
-		}, "Could not get endpoints", &helpers.TimeoutConfig{Timeout: 100})
+		}, "could not get endpoints", &helpers.TimeoutConfig{Timeout: 100})
 
 		Expect(epsStatus).Should(BeNil())
 		appPods = getAppPods()
 
-		endpoints, err := kubectl.CiliumEndpointsListByTag(ciliumPod, podFilter)
+		endpoints, err := kubectl.CiliumEndpointsListByLabel(ciliumPod, podFilter)
 		policyStatus := endpoints.GetPolicyStatus()
-		Expect(policyStatus["enabled"]).Should(Equal(2))
-		Expect(policyStatus["disabled"]).Should(Equal(2))
+		Expect(policyStatus[helpers.Enabled]).Should(Equal(2))
+		Expect(policyStatus[helpers.Disabled]).Should(Equal(2))
 
 		trace := kubectl.CiliumExec(ciliumPod, fmt.Sprintf(
 			"cilium policy trace --src-k8s-pod default:%s --dst-k8s-pod default:%s --dport 80",
-			appPods["app2"], appPods["app1"]))
+			appPods[helpers.App2], appPods[helpers.App1]))
+
 		trace.ExpectSuccess(trace.Output().String())
 		Expect(trace.Output().String()).Should(ContainSubstring("Final verdict: ALLOWED"))
 
 		trace = kubectl.CiliumExec(ciliumPod, fmt.Sprintf(
 			"cilium policy trace --src-k8s-pod default:%s --dst-k8s-pod default:%s",
-			appPods["app3"], appPods["app1"]))
+			appPods[helpers.App3], appPods[helpers.App1]))
 		trace.ExpectSuccess()
 		Expect(trace.Output().String()).Should(ContainSubstring("Final verdict: DENIED"))
 
 		_, err = kubectl.Exec(
-			"default", appPods["app2"], fmt.Sprintf("curl http://%s/public", clusterIP))
+			helpers.DefaultNamespace, appPods[helpers.App2], fmt.Sprintf("curl http://%s/public", clusterIP))
```

Comment:
Why do you not use `CurlFail` here?

Reply:
I only searched for curl commands which exactly matched `curl -s --fail --connect-timeout %d %s`. I will see if I can wrap all curl commands with `CurlFail`.

Reply:
Would you be OK with me adding this in a subsequent PR? I can file a ticket for this. I'd like to get this merged as soon as possible.